### PR TITLE
Footer component

### PR DESCRIPTION
**Pull request by JakubBrozek**
# Tasks 
- Adding footer on page
- Adding icons
- Adding links to socialmedia
- Styling `Footer` and it's elements

# Steps 
- [x] Added `Footer` component
- [x] Styled `footer-wrapper` and icons
- [x] Activated links to socialmedia
- [x] Added footer line with gradient
- [x] Added client's logo in the background
- [x] Added `credits` with year 

# Preview
image

### CODE REVIEW

File: src/components/footer.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import '../fontawesome/css/all.min.css';
+import '../fontawesome/css/fontawesome.min.css';
+
+const Footer = () => (
+  <li className='footer-wrapper'>
+    <div className='footer-logo'>
+      <a href="http://mikstura.it" className='footer-link'>mikstura<span className="header-link__it">.IT</span></a>
+    </div>
+    <ol className='footer__icon'>
+      <a href="https://www.meetup.com/Mikstura-IT/" className='footer-link' activeClassName='active'><i class="fab fa-meetup"></i></a>
```
Comment (wozny1989) [inline src/components/footer.js]: I guess `activeClassNames` it not necessary in this cases (in all social links) ✂️

File: src/fontawesome/LICENSE.txt [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+We've kept attribution comments terse, so we ask that you do not actively work
+to remove them from files, especially code. They're a great way for folks to
+learn about Font Awesome.
+
+# Brand Icons
+All brand icons are trademarks of their respective owners. The use of these
+trademarks does not indicate endorsement of the trademark holder by Font
+Awesome, nor vice versa. **Please do not use brand logos for any purpose except
+to represent the company, product, or service to which they refer.**
```
Comment (wozny1989) [inline src/fontawesome/LICENSE.txt]: Remove this file ✂️

File: src/styles/stylesheet.css [hunk 3]
```diff
@@ -6,6 +6,24 @@ body {
...
+
+.footer__icon {
+  border-style: solid;
+  border-color: #404040;
+  border-radius: 50%;
+  padding: 5px 10px;
+  margin: 0;
+  display: flex;
+  justify-content: space-evenly;
+}
```
Comment (wozny1989) [inline src/styles/stylesheet.css]: Consider set static `width` and `height` to make sure we have square, not an oval. Also set after this `aling-items: center` to align icons in squares.

File: src/styles/stylesheet.css [hunk 4]
```diff
@@ -6,6 +6,24 @@ body {
   --primary-color: orange;
 }
 
+.footer-wrapper {
+  width: 60%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 22px;
+  margin: 0 auto;
```
Comment (wozny1989) [inline src/styles/stylesheet.css]: Think about to replace `0` to some value to make a little vertical margin.

File: src/styles/stylesheet.css [hunk 5]
```diff
@@ -59,11 +91,21 @@ body {
   color: var(--primary-color)
 }
 
+.footer-link.active{
```
Comment (wozny1989) [inline src/styles/stylesheet.css]: `.active` ✂️

**Review by wozny1989 — CHANGES_REQUESTED** [review]
Great pull request documentation, keep it up! 🎉 🎉 🎉

File: src/styles/stylesheet.css [hunk 6]
```diff
@@ -6,6 +6,27 @@ body {
...
+.footer__icon {
+  width: 45px;
+  height: 45px;
+  border-style: solid;
+  border-color: #404040;
+  border-radius: 50%;
+  padding: 5px 10px;
+  margin: 0;
+  display: flex;
+  justify-content: space-evenly;
```
Comment (wozny1989) [inline src/styles/stylesheet.css]: Maybe better to use `center`

File: src/styles/stylesheet.css [hunk 7]
```diff
@@ -6,6 +6,27 @@ body {
...
+  align-items: center;
+  font-size: 22px;
+  margin: 30px auto;
+}
+
+.footer__icon {
+  width: 45px;
+  height: 45px;
+  border-style: solid;
+  border-color: #404040;
```
Comment (wozny1989) [inline src/styles/stylesheet.css]: You use several times the same color `#404040`. Consider set it in variable.

File: src/components/layout.js [hunk 8]
```diff
@@ -32,6 +32,7 @@ const Layout = ({ children }) => {
       </div>
       <div className='content'>
         {children}
+        <Footer />
```
Comment (wozny1989) [inline src/components/layout.js]: Maybe better way will be move that `<Footer />` below `content` container and set width. Just for make some empty place you can set in `content` some static height and then you can get rid of `align-items: flex-end` in `content` class. (`flex-end` will be undesirable behavior when you pass some data to `Layout` component)

File: src/components/footer.js [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+      </ol>
+      <ol className='footer__icon'>
+        <a href="https://linkedin.com/company/mikstura-it-foundation" className='footer-link'><i class="fab fa-linkedin"></i></a>
+      </ol>
+      <ol className='footer__icon'>
+        <a href="mailto:fundacja@mikstura.it" className='footer-link' ><i class="far fa-envelope"></i></a>
+      </ol>
+    </li>
+    <div className='footer__bottomline'></div>
+    <div className='footer-credits'>© {new Date().getFullYear()} Selleo</div>
```
Comment (wozny1989) [inline src/components/footer.js]: Looks great! 🎉

**Review by wozny1989 — CHANGES_REQUESTED** [review]
@JakubBrozek Some small feedback 👏

File: src/styles/stylesheet.css [hunk 10]
```diff
@@ -4,6 +4,7 @@
 body {
   font-family: 'Poppins', sans-serif;
   --primary-color: orange;
+  --secondary-color: #404040;
```
Comment (wozny1989) [inline src/styles/stylesheet.css]: ❤️

Comment (JakubBrozek) [reply src/styles/stylesheet.css]: We're on fire 👯

File: src/components/layout.js [hunk 11]
```diff
@@ -33,6 +33,7 @@ const Layout = ({ children }) => {
       <div className='content'>
```
Comment (wozny1989) [inline src/components/layout.js:33]: Use bootstrap class `container` (to make a responsive)

File: src/styles/stylesheet.css [hunk 12]
```diff
@@ -20,8 +27,10 @@ body {
 }
 
 .content {
-  background-color: white;
-  color: #404040;
+  margin: 0 auto;
+  width: 1140px;
+  height: 650px;
+  color: var(--secondary-color);
 }
```
Comment (wozny1989) [inline src/styles/stylesheet.css]: Change class name to `.container` and remove static `width` field. Look at the behavior page in different size.

File: src/components/layout.js (hunk 11 above)
Comment (wozny1989) [inline src/components/layout.js]: Use bootstrap class `container` (to make a responsive)

**Review by wozny1989 — CHANGES_REQUESTED** [review]

**Pull request merged**